packages/api/lib/writeRecords/write-granules.js

```diff
@@ -468,23 +457,21 @@ const _writeGranuleRecords = async (params) => {
     apiGranuleRecord,
     knex,
     esClient = await Search.es(),
-    granuleModel,
     executionCumulusId,
     granulePgModel,
   } = params;
   let pgGranule;
   log.info('About to write granule record %j to PostgreSQL', postgresGranuleRecord);
-  log.info('About to write granule record %j to DynamoDB', apiGranuleRecord);
-
   try {
     await createRejectableTransaction(knex, async (trx) => {
+      // Validate API schema
+      recordIsValid(apiGranuleRecord, granuleSchema, false);
```

Comment:
This is needed as we're no longer using the granule model write logic, but we should validate the API schema on write in addition to the underlying datastore schema validation.